### ISSUE TITLE
For January 2019

### ISSUE THREAD

**Issue by HarryMacfinned** [bisq-network/compensation#217]
## Summary

> _Specify the total amount of BSQ you are requesting, and the BSQ address that amount should be paid to:_

 - BSQ requested: `2400`
 - BSQ address: B19yxYCRTVaDfRfucrH6aEzeZpoVTv8Je8S

This compensation request :
- follows the https://docs.bisq.network/roles.html#compensation recommendations
- applies the 1BSQ=1USD rule
- is atm quite a copy/paste from the previous compensation requests (for the roles) which didn't take into account the activity increase since Q4/2018 (-> this will probably be updated next month).

### Substantive reports

- Support |  | (BSQ  requested = 2000)
Evaluation is based and intended in line with : previous compensation requests, https://github.com/bisq-network/compensation/issues/24 , and other support compensation requests.

Most questions are asked on the forum were the number of posts varied from 265 in july 2018 to 353 this month. The volume of posts increased roughly in the same way on the slack and the github. (On the same period, the number of trades varied from 692 to 2300.)
Accordingly, the time needed in january 2019 to assume the support role increased (significantly) compared to july 2018 when I began with this role.
If the trend still stays the same next month, I'll update my request accordingly.

- Fees and connectivity monitoring | see the weekly reports in https://github.com/bisq-network/roles/issues/10 | (BSQ requested = 200)
TL;DR: there was zero catch this month (nodes are running well).

- Prices monitoring | https://github.com/bisq-network/roles/issues/10#issuecomment-459427778 | (BSQ requested = 200)
TL;DR: There were no noticeable divergences between pricenodes this month.

## Other points (BSQ requested = 0)
- helped with v0.9.2 and v0.9.3 release
- investigated seednode issue https://github.com/bisq-network/roles/issues/15#issuecomment-453749038 2019-01-12
- Clearnet Bisq webpages monitoring : 1 alert this month https://bisq.network/markets/ 2019-01-29

## Contributions in progress
- I'm working on a support documentation (compilation of main support issues)

**Comment by ripcurlx** [comment]
Closing as complete, see https://github.com/bisq-network/compensation/issues/204#issuecomment-460541588.
